### PR TITLE
Decouple v2 webui

### PR DESCRIPTION
**Pull request by stevezau**
### Motivation for changes:

Start the process to de-couple the WebUI into https://github.com/Flexget/webui

### TODO

- [x]  Remove v2 React Code
- [x]  Cleanup handling when running from git
- [x]  Add readme's in each webui version
- [x]  Fix Release in Jenkins

### CODE REVIEW

**Comment by kristenmills** [comment]
I think you forgot to remove the stuff from the .travis.yml related to v2. That's why the travis build is failing

File: flexget/ui/v1/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import os
 import fnmatch
 
-from flask import send_from_directory, Flask
+from flask import send_from_directory, Flask, abort
```
Comment (liiight) [inline flexget/ui/v1/__init__.py]: why add `abort`?

Comment (stevezau) [reply flexget/ui/v1/__init__.py]: Ah, i was using it to send a 404 but figured out a different way to handle running from github. Removed it.

File: flexget/ui/v2/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Failed to load Flexget Web UI</title>
+</head>
+<body>
+<h1>Failed to load Flexget Web UI</h1>
+
+<p>It looks like you are running flexget from the GitHub source. If you want to use the webui you'll have to run the WebUI build steps by following
+    <a href="http://flexget.com/wiki/Web-UI">http://flexget.com/wiki/Web-UI/v2</a></p>
```
Comment (kristenmills) [inline flexget/ui/v2/index.html]: href points to the v1 wiki. Should be changed to v2

Comment (kristenmills) [reply flexget/ui/v2/index.html]: Also I need to update the v2 wiki. I'll do that today.

File: flexget/ui/v2/__init__.py [hunk 3]
```diff
@@ -14,10 +14,11 @@
 
 manager = None
 debug = False
-app_base = None
+exists = True
 
 ui_base = os.path.dirname(os.path.realpath(__file__))
```
Comment (kristenmills) [inline flexget/ui/v2/__init__.py]: Can we make this configurable? If someone wants to run flexget from git, I don't think the solution should be to copy the ui folders. Maybe have some way to specify in the config where the dist folder is located?

Comment (stevezau) [reply flexget/ui/v2/__init__.py]: You can always setup a symlink. If you are really doing dev work on the WebUI you would run it separately anyway.

**Pull request merged**